lib/ui/search_user/usecases/search_for_user_use_case.dart

```diff
@@ -1,22 +1,20 @@
-
 import 'package:hypha_wallet/core/error_handler/model/hypha_error.dart';
+import 'package:hypha_wallet/core/network/api/services/remote_config_service.dart';
 import 'package:hypha_wallet/core/network/models/user_profile_data.dart';
 import 'package:hypha_wallet/core/network/repository/user_account_repository.dart';
 import 'package:hypha_wallet/ui/architecture/result/result.dart';
 import 'package:hypha_wallet/ui/shared/usercases/get_user_profiles_from_accounts_use_case.dart';
 
-
 class SearchForMemberUseCase {
   final UserAccountRepository userAccountRepository;
   final GetUserProfilesFromAccountsUseCase getUserProfilesFromAccountsUseCase;
-  // FindAvailableAccountUseCase
 
   SearchForMemberUseCase(this.userAccountRepository, this.getUserProfilesFromAccountsUseCase);
 
   Future<Result<List<UserProfileData>, HyphaError>> run(String searchQuery) async {
     final Result<List<String>, HyphaError> result = await userAccountRepository.findHyphaAccounts(searchQuery);
-    if(result.isValue) {
-      return getUserProfilesFromAccountsUseCase.run(result.asValue!.value);
+    if (result.isValue) {
+      return getUserProfilesFromAccountsUseCase.run({Network.eos: result.asValue!.value});
```

Comment:
Nik. this is very very important. needs your attention: 

 await userAccountRepository.findHyphaAccounts(searchQuery);

we make a call to get `findHyphaAccounts`

I asumme the findHyphaAccounts are all in the Network.eos ?? and i am setting those accounts to use that network

Reply:
Actually in this case it should use the network of the user

Ie, if I am logged in / active as "illumination" on Telos, then this find function should find people on the same network, ie, Telos

If I am logged in as "illum1nation" on EOS, then find should find people on EOS ... 

We can't send coins from EOS to Telos & vice versa... at least not easily, although we may add cross chain transactions later, but I think we will add them only for the Hypha token...